multi/imxrt-multi/imxrt-multi.h

```diff
@@ -79,27 +85,35 @@ typedef struct {
 	};
 
 } spi_t;
+/* clang-format on */
+
+
+/* Cortex M4 */
 
 
+#define CM4_LOAD_FILE 0
+#define CM4_LOAD_BUFF 1
+#define CM4_RUN_CORE  2
+
 
 /* MULTI */
 
 
-typedef struct _multi_i_t {
+typedef struct {
 	id_t id;
 
 	union {
 		gpio_t gpio;
 		spi_t spi;
+		int cm4_type;
```

Comment:
cm4 is only valid for the imxrt1170 target, maybe something like:
```suggestion
#ifdef TARGET_IMXRT1170
		int cm4_type;
#endif
```
btw. a similar check would be good for all `#define CM4_*` that are specific to the target.

Reply:
Doesn't really hurt anything, I'd rather keep one, coherent API